test/test_client.py

```diff
@@ -731,47 +733,6 @@ def test_revisions(self):
         assert revisions[1]['revid'] == 689816909
 
 
-class TestVersionTupleFromGenerator:
```

Comment:
er, I don't think we want to completely remove a test case? I assume this is a mistake from rebasing on your version parsing improvement PR...

Reply:
Yep, thank you for catching that.